client/src/components/header/Header.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styled from "styled-components";
+
+const Header = () => {
+    return <StyledHeader>BockWurst</StyledHeader>;
+  };
```

Comment:
You could pass a prop here, to make the `Header` reusable.
```suggestion
const Header = ({label}) => {
    return <StyledHeader>{label}</StyledHeader>;
  };
```

Reply:
Done, thank you!